backend/spvm.pm

```diff
@@ -0,0 +1,108 @@
+# Copyright © 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package backend::spvm;
+use strict;
+use base ('backend::virt');
+use testapi qw(get_var get_required_var check_var);
+
+use IO::Select;
+
+# supporting the minimal command set of NovaLink through a ssh tunnel
+
+sub new {
+    my $class = shift;
+    my $self  = $class->SUPER::new;
+    get_required_var('WORKER_HOSTNAME');
+
+    return $self;
```

Comment:
No so important, but it could be simplify like this: `return $class->SUPER::new;` (seen in others backend).

Reply:
It would end up being the same.